CHANGELOG

```diff
@@ -1,4 +1,30 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate positional arguments in the following methods:
```

Comment:
Please reword this like:

Deprecate specifying keyword arguments as positional arguments in the public methods:

This will become a hard failure in Nmstate-0.3.0 and later.

Reply:
Done, thanks!